### DISCUSSION TITLE
[Question] Telemetry

### DISCUSSION THREAD

**Discussion by ayushsharma82** [lamassuiot/lamassu-compose#11] [General]

**Comment by ayushsharma82** [comment]
Hi guys, 

Awesome PKI approach for IoT. 

Q: Do you have any telemetry enabled in lamassu?

We usually get clients from national defence department which are concerned about security. Knowing about any telemetry ( anonymous or not ) sent to any server is crucial. 

Regards,
Ayush

**Reply by jjrodrig** [reply]
Hi @ayushsharma82

Thanks for your interest in our project. 

There is not any service that sends data to external servers if that is your concern (Except for Cloud integration). It can run in an offline environment. 

We are using jaeger for distributed calls tracing but not external reporting is included in our platform. 

Regards, 
Juanjo

**Reply by mikelamutxastegi** [reply]
Hi @ayushsharma82,

Thanks for your question. We are great to hear about your interest on Lamassu.

In addition to the Jaeger service for distributed calls tracing mentioned by @jjrodrig, each of the Lamassu services reports some metrics in the /metrics endpoint that can be consumed by a monitoring service, such as, [Prometheus](https://prometheus.io/docs/introduction/overview/).

Do not hesitate to ask if you need more details.

Best regards,
Mikel.
